cmd/zoekt-sourcegraph-indexserver/cleanup.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+	"sort"
+	"time"
+
+	"github.com/google/zoekt"
+)
+
+func cleanup(indexDir string, repos []string, now time.Time) {
+	trashDir := filepath.Join(indexDir, ".trash")
+	if err := os.MkdirAll(trashDir, 0755); err != nil {
+		log.Printf("failed to create trash dir: %v", err)
```

Comment:
Should we abort here?

Reply:
nope, we will fallback to `os.Remove` on items that don't exist in this rare case.